Apps/WebClient/src/ClientApp/src/components/timeline/calendarTimeline.vue

```diff
@@ -52,14 +52,20 @@ export default class CalendarTimelineComponent extends Vue {
     @Prop() private totalEntries!: number;
     @Prop() private filterText!: string;
     @Prop() private filterTypes!: string[];
+    @Prop() private allFilterTypes!: string[];
 
     private filteredTimelineEntries: TimelineEntry[] = [];
 
     @Watch("filterText")
     @Watch("filterTypes")
     private applyTimelineFilter() {
         this.filteredTimelineEntries = this.timelineEntries.filter((entry) =>
-            entry.filterApplies(this.filterText, this.filterTypes)
+            entry.filterApplies(
```

Comment:
this logic can happen directly on timeline since is the same for both types of timelines. no matter now tho, I will refactor this section with the filters component